examples/mock-rock/1.2/rockcraft.yaml

```diff
@@ -6,11 +6,11 @@ base: bare
 build-base: ubuntu@22.04
 license: Apache-2.0
 platforms:
-  ppc64el:
+  # ppc64el:
```

Comment:
why is this removed?

Reply:
This is to temporarily avoid lpci builds, since they are always failing.